docs/en/api/generated/mmedit.apis.delete_cfg.rst

```diff
@@ -0,0 +1,6 @@
+﻿mmedit.apis.delete\_cfg
```

Comment:
Don't upload/commit `api/generated` directory, like https://github.com/open-mmlab/mmengine/pull/469/files .

Reply:
Ok, I will remove these in the next commit